tests/Offer/CommandHandlers/RemoveLabelHandlerTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace CultuurNet\UDB3\Offer\CommandHandlers;
+
+use Broadway\CommandHandling\Testing\CommandHandlerScenarioTestCase;
+use Broadway\EventHandling\EventBus;
+use Broadway\EventStore\EventStore;
+use CultuurNet\UDB3\Calendar;
+use CultuurNet\UDB3\CalendarType;
+use CultuurNet\UDB3\Event\EventRepository;
+use CultuurNet\UDB3\Event\Events\EventCreated;
+use CultuurNet\UDB3\Event\Events\LabelAdded;
+use CultuurNet\UDB3\Event\Events\LabelRemoved;
+use CultuurNet\UDB3\Event\EventType;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Label;
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Offer\Commands\RemoveLabel;
+use CultuurNet\UDB3\Offer\OfferRepository;
+use CultuurNet\UDB3\Place\PlaceRepository;
+use CultuurNet\UDB3\Title;
+
+final class RemoveLabelHandlerTest extends CommandHandlerScenarioTestCase
+{
+    protected function createCommandHandler(EventStore $eventStore, EventBus $eventBus): RemoveLabelHandler
+    {
+        return new RemoveLabelHandler(
+            new OfferRepository(
+                new EventRepository($eventStore, $eventBus),
+                new PlaceRepository($eventStore, $eventBus)
+            )
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_should_remove_labels_previously_added_even_if_the_visibility_is_different(): void
+    {
+        $id = '4c6d4bb8-702b-49f1-b0ca-e51eb09a1c19';
+
+        $this->scenario
+            ->withAggregateId($id)
+            ->given(
+                [
+                    $this->eventCreated($id),
+                    new LabelAdded($id, new Label('foo', true)),
```

Comment:
We also need to test the removal of a label with the same visibility?

Reply:
I figured that would also work then, but here's an additional test 😉 94aeb27